modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -486,6 +486,12 @@ external to the cluster and from all the nodes within the cluster.
 machines that run the Ingress router pods, which are the worker nodes by
 default. This record must be resolvable by both clients external to the cluster
 and from all the nodes within the cluster.
+
+|Kubernetes API
+|`api-int.<cluster_name>.<base_domain>.`
```

Comment:
@sgaoshang and @jcpowermac .

Each record name ends with a period (.), this is correct?